modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ViewTranslationDisplayContext.java

```diff
@@ -132,14 +131,14 @@ public List<String> getStringValues(InfoField infoField, Locale locale) {
 		Collection<InfoFieldValue<Object>> infoFieldValues =
 			infoItemFieldValues.getInfoFieldValues(infoField.getUniqueId());
 
-		Stream<InfoFieldValue<Object>> stream = infoFieldValues.stream();
+		List<String> stringValues = new ArrayList<>();
 
-		return stream.map(
-			infoFieldValue -> GetterUtil.getString(
-				infoFieldValue.getValue(locale))
-		).collect(
-			Collectors.toList()
-		);
+		for (InfoFieldValue<Object> infoFieldValue : infoFieldValues) {
+			stringValues.add(
+				GetterUtil.getString(infoFieldValue.getValue(locale)));
+		}
+
+		return stringValues;
```

Comment:
Can you use a regular loop here? See https://issues.liferay.com/browse/LPS-150112